client/js/widgets/widget.js

```diff
@@ -984,20 +984,26 @@ export class Widget extends StateManaged {
       }
 
       if(a.func == 'FLIP') {
-        setDefaults(a, { count: 0, face: null, faceCyle: null, collection: 'DEFAULT' });
+        setDefaults(a, { count: 'all', face: null, faceCyle: null, collection: 'DEFAULT' });
+        if(a.count === 'all')
+          a.count = 999999;
+        
+        if(typeof(a.count)!="number")
```

Comment:
I guess I would do something like (here and in the other functions)
```
a.count = parseInt(a.count);
if (isNaN(a.count))
  a.count = 0
  <log problem>
```
Note that as it is, non-integer numbers will be accepted.

Reply:
> Note that as it is, non-integer numbers will be accepted.

But it does work. it just rounds down. I don't want to change previous behavior

I do agree I should change the count value

Reply:
That sounds right.

Reply:
@rogerl50 done. please check if everything is resolved